src/internal/analytics-metadata/attributes.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { GeneratedAnalyticsMetadataFragment } from './interfaces';
+
+export const METADATA_DATA_ATTRIBUTE = 'awsuiAnalytics';
+export const METADATA_ATTRIBUTE = `data-awsui-analytics`;
+export const LABEL_DATA_ATTRIBUTE = 'awsuiAnalyticsLabel';
+const LABEL_ATTRIBUTE = 'data-awsui-analytics-label';
+
+export const getAnalyticsMetadataAttribute = (metadata: GeneratedAnalyticsMetadataFragment) => ({
```

Comment:
Where is this function expected to be called? Is the caller expected to handle any errors from JSON.stringify if a malformed object is passed in?

Reply:
It will be called in the components repo, for example: https://github.com/cloudscape-design/components/commit/35ea770e181a9187c2b6226e8467a82bc9ddc944#diff-b8b610715507e7a442c581cf34b7f6949332b7395da79e61287695f94dc7b72eR71

What would be an example of malformed object?

Reply:
Couldn't think of any 😅 So I guess this is probably fine then.